### PR TITLE
6502: pseudo disassembler

### PR DESCRIPTION
**Pull request by q6r**
6502 pseudo disassembler. Need more instructions. Need improvement for indirect indexed and indexed indirect instructions.

### CODE REVIEW

File: libr/parse/p/parse_6502_pseudo.c [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+        if (ptr) {
+            *ptr = '\0';
+			for (++ptr; *ptr==' '; ptr++);
+            strncpy (w0, buf, sizeof (w0) - 1);
+            strncpy (w1, ptr, sizeof (w1) - 1);
+            optr = ptr;
+            ptr = strchr (ptr, ',');
+            if (ptr) {
+                *ptr = '\0';
+				for (++ptr; *ptr==' '; ptr++);
```
Comment (crowell) [inline libr/parse/p/parse_6502_pseudo.c]: indentation

File: libr/parse/p/parse_6502_pseudo.c [hunk 2]
```diff
@@ -0,0 +1,196 @@
+/* radare - LGPL - Copyright 2015 - pancake */
+
```
Comment (crowell) [inline libr/parse/p/parse_6502_pseudo.c]: this whole file should use tabs, not spaces

also copyright yourself!

File: libr/parse/p/parse_6502_pseudo.c [hunk 3]
```diff
@@ -0,0 +1,196 @@
+/* radare - LGPL - Copyright 2015 - pancake */
```
Comment (crowell) [inline libr/parse/p/parse_6502_pseudo.c]: copyright 2015 - qnix

**Comment by radare** [comment]
Remove the struct fields filled with NULL (not necessary).

I prefer to use the same syntax for all pseudodisassemblers: use [] instead of * or ().

Are you using two-space indent? Use tabs!

**Comment by radare** [comment]
Merged!

File: libr/parse/p/parse_6502_pseudo.c [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+
+typedef enum {
+	IND_IDX = 0,
+	IDX_IND = 1,
+	NORM = 2,
+} ADDR_TYPE;
+
+static int replace(int argc, const char *argv[], char *newstr, ADDR_TYPE type) {
+	int i, j, k;
+	struct {
```
Comment (jvoisin) [inline libr/parse/p/parse_6502_pseudo.c]: Isn't this struct `const`?

**Pull request closed without merging**